### PR TITLE
Angular init

### PR DESCRIPTION
**Pull request by XQuader**
First commit with angular @NE05KY @strellka

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+<head>
+  <meta charset="UTF-8">
+  <title>Login page</title>
+  <script src="http://ajax.googleapis.com/ajax/libs/angularjs/1.5.0-rc.1/angular.min.js"></script>
+  <script src="js/app.js"></script>
+</head>
+<body ng-app="loginPage">
+	<form ng-submit="login.submit(info.name)">
+		<div><input type="textarea" ng-model="info.name"\>
+		<div><input type="password" ng-model="info.password"\>
```
Comment (strellka) [inline index.html]: <div> tags are not closed, fix it please

File: README.md [hunk 2]
```diff
@@ -1,2 +1,2 @@
-# itlabsctrl
-Система организации проведения и проверки лабораторных работ по программированию
+Релиз расположен по адресу http://xquader.github.io/itlabsctrl/
+Система организации проведения и проверки лабораторных работ по программированию (Front-end)
```
Comment (strellka) [inline README.md]: Russian shouldn't be used. Translate it to English.

File: index.html (hunk 1 above)
Comment (XQuader) [inline index.html]: Ok. I'll do it tomorrow

**Comment by strellka** [comment]
approved.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,16 @@
+<!doctype html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <title>Login page</title>
+  <script src="http://ajax.googleapis.com/ajax/libs/angularjs/1.5.0-rc.1/angular.min.js"></script>
```
Comment (NE05KY) [inline index.html]: Don't specify protocol for outer resources
see http://www.paulirish.com/2010/the-protocol-relative-url/

File: index.html [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+  <script src="js/app.js"></script>
+</head>
+<body ng-app="loginPage">
```
Comment (NE05KY) [inline index.html]: Are you sure this is name for your application? It looks like module name

Comment (XQuader) [inline index.html]: It was pushed "as is" only for pullrequest test, i know what to fix).

File: index.html (hunk 3 above)
Comment (XQuader) [inline index.html]: Thanks, didn't know that.

**Comment by NE05KY** [comment]
approved

**Pull request merged**